### PR TITLE
Implemented and tested  Cholesky for Paddel frontends

### PR DESCRIPTION
**Pull request by XinyuanWang283**
Close https://github.com/unifyai/ivy/issues/16464

### CODE REVIEW

**Comment by XinyuanWang283** [comment]
@rishabgit Could you please help me check my code?

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_tensor.py
Comment (rishabgit) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_tensor.py:1]: shouldn't replace existing unrelated tests

**Review by rishabgit — CHANGES_REQUESTED** [review]
Hi! 😄
Please refer to the docs on how to pick up an issue that your PR gets correctly linked - https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues

Also, looks like you're accidentally removing other tests 😅. Could you please look into this too?

**Comment by XinyuanWang283** [comment]
Close #16464

**Comment by XinyuanWang283** [comment]
@rishabgit,  Could you please help me check my code one more time? Thank you!

**Comment by XinyuanWang283** [comment]
@rishabgit Could you review my code again? It has been a week. Thank you for your help.

**Comment by rishabgit** [comment]
Hi @XinyuanWang283, sorry about the delay. There were slight lint issues, which I've updated to get the test passing. In future, please use [pre-commit](https://unify.ai/docs/ivy/overview/contributing/setting_up.html#pre-commit) to check for the lint errors before a commit. Besides this, it seems to pass the tests and everything LGTM! 
Thanks a ton! 😄

**Comment by XinyuanWang283** [comment]
@rishabgit Wow! Thank you very much for your help and kind reminder! I will use pre-commit in the future!

**Pull request merged**